pkgs/misc/cups/drivers/hp-uld/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib, stdenv, fetchurl, cups, libusb-compat-0_1, libxml2 }:
+
+let
+arch = if stdenv.hostPlatform.system == "x86_64-linux"
+  then "x86_64"
+  else "i386";
+in stdenv.mkDerivation rec {
+  pname = "hp-uld";
+  version = "1.00.39.12_00.15";
+
+  src = fetchurl {
```

Comment:
maybe I am missing something, but why is this `fetchurl` not `fetchzip` (which despite the name unpacks a number of different archive types)